rootfs-builder/rootfs.sh

```diff
@@ -141,7 +141,7 @@ generate_dockerfile()
 	[ -n "$http_proxy" ] && readonly set_proxy="RUN sed -i '$ a proxy="$http_proxy"' /etc/dnf/dnf.conf /etc/yum.conf; true"
 
 	readonly install_go="
-RUN cd /tmp ; curl -OL https://storage.googleapis.com/golang/go${GO_VERSION}.linux-${goarch}.tar.gz
+RUN cd /tmp ; curl -x $http_proxy -OL https://storage.googleapis.com/golang/go${GO_VERSION}.linux-${goarch}.tar.gz
```

Comment:
does this work if `$http_proxy` is not defined?, which might be the case for some folks.
If so, lgtm - otherwise, we should make it add that in only if it is set. I think the notion of the patch is good though :-)
/cc @chavafg @jcvenegas

Reply:
I'd propose having a env variable named e.g. `HTTP_PROXY`, that when detected just adds the `-x $HTTP_PROXY` option to curl:
```
curlOptions=("-OL")
[ -n "$HTTP_PROXY" ] && curlOptions+=("-x $HTTP_PROXY")
curl "${curlOptions[@]}" https://...
```

Reply:
@marcov updated 👍